src/index.js

```diff
@@ -14,6 +15,7 @@ class CompressionPlugin {
     this.compressionOptions = {};
 
     if (typeof this.algorithm === 'string') {
+      // eslint-disable-next-line global-require
```

Comment:
Please always move `eslint-disable` stuff to the top of the particular file :)

```js
/* eslint-disable
* global-require,
* more rules...,
*/
```

Reply:
@michael-ciniawsky i think it is bad practice because disable rule on all file can provoke errors in places where I would not want to ignore them. Disabled rule should be on line (before line) where i want disable, not on all file.

Reply:
@evilebottnawi You can also leave them it's not a critical issue by any means, but I would still argue that while you are right for things like `eval` `no-eval` should occur where the code is, styling related stuff should be at the top (if needed) to lessen the noise while reading the code. These comments shouldn't be necessary at all normally... :)